app/javascript/components/supplemental_application/sections/LeaseInformatonInputs.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'
+import { Select, Text } from 'react-form'
+import { map, toSafeInteger } from 'lodash'
+
+import FormGrid from '~/components/molecules/FormGrid'
+import { pluck, decorateComponents } from '~/utils/utils'
+import { withNestedForm } from '~/utils/form/hoc'
+import formUtils from '~/utils/formUtils'
+import { withContext } from '../context'
+
+const Plain = ({ text }) => <div>{text}</div>
+
+const INPUTS = { 'Select': Select, 'Text': Text, 'Plain': Plain }
+
+const CustomFormGrid = decorateComponents(INPUTS, Component => {
+  return ({ label, ...rest }) => (
+    <FormGrid.Item>
+      <FormGrid.Group label={label}>
```

Comment:
does each of the fields need to be a formgrid group? I'm not sure this added complexity is helpful here.

Reply:
This is how jesse structure the html. it' using some form-group classes that are adding some styling. Without the group it will look different in the UI